spec/acceptance/00_backend_spec.rb

```diff
@@ -13,14 +13,81 @@
       apply_manifest_on(node, pp, :catch_changes  => true)
     end
 
-    it do
-      on node, "/opt/puppetlabs/bin/facter os" do
-        puts stdout
+    describe service('sensu-backend'), :node => node do
+      it { should be_enabled }
+      it { should be_running }
+    end
+  end
+
+  context 'with SSL' do
+    it 'should bootstrap certs' do
+      on node, 'mkdir -p /root/ssl'
+      on node, 'openssl req -newkey rsa:2048 -nodes -keyout /root/ssl/ca_key.pem -x509 -days 365 -out /root/ssl/ca.pem -subj "/CN=localhost"'
```

Comment:
We could just copy files that are checked into this repo that correspond with puppet certs. How are we checking that an agent on the other non-backend node can talk SSL?

Reply:
I think I'm going to move the SSL tests to a separate file that comes after the basic backend/agent tests so we can do end-to-end tests that SSL for both backend and agent works.

Can't copy the files because when beaker copies the module to nodes it excludes tests and spec directories.  Also the changes I'm making per these comments are in #1027.